gitter/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+
+
+class Question(models.Model):
+    identifier = models.CharField(max_length=500, primary_key=True)
+    room = models.CharField(max_length=300)
+    text = models.TextField()
+    sent_at = models.DateTimeField()
+    sent_by = models.CharField(max_length=300)
+
+    def __str__(self):
+        return ('sent_by: ' + self.sent_by + ' on room: ' + self.room)
+
+
+class Answer(models.Model):
+    identifier = models.CharField(max_length=500, primary_key=True)
+    room = models.CharField(max_length=300)
+    text = models.TextField()
+    sent_at = models.DateTimeField()
+    sent_by = models.CharField(max_length=300)
+
+    def __str__(self):
+        return ('sent_by: ' + self.sent_by + ' on room: ' + self.room)
```

Comment:
the `identifier` and `text` are more useful

also add `__repr__` to be more helpful thant `__str__` for debugging